app.json

```diff
@@ -59,17 +59,6 @@
       "required": true
     },
 
-    "TWILIO_MESSAGE_SERVICE_SID": {
```

Comment:
This differs from the changes to `.env.example`. Was this intended to remove `TWILIO_MESSAGE_CALLBACK_URL` instead, which was never defined in `app.json`?